src/hooks/useOktaAxios.js

```diff
@@ -8,7 +8,7 @@ export default function useOktaAxios() {
   return axios.create({
     baseURL: process.env.REACT_APP_BACKENDURL || '',
     headers: {
-      authorization: isAuthenticated ? `${idToken.value}` : '',
+      authorization: isAuthenticated ? `Bearer ${idToken.value}` : '',
     },
```

Comment:
Very happy to have this fixed, it was vital for the this program to function correctly. Was able to test and verify code work and fixed issue with not displaying incidents